nat-lab/tests/utils/batching.py

```diff
@@ -0,0 +1,161 @@
+# There's a dillemma in how to best capture the traffic. Capturing on the node itself
+# provides insights into the packets themselves coming to and from the tunnel device,
+# however capturing locally doesn't mean the packet actually left the machine.
+# An alternative is to capture on gateways which guarantees the packet arrival
+# however there's no way to easily pry open the packets and observe their insides.
```

Comment:
I guess it would be nice if this comment would actually say which way of capturing them is implemented here.

Reply:
I though its a pretty nice disclaimer inside of `batching.py` however making promises there about how we use it is pretty wrong since it's disconnected and the file contains general purpose code. As a middle ground I moved the comment inside of the testcase itself and mentioned that both are used.